tests/flow/test_concurrent_query.py

```diff
@@ -331,3 +332,22 @@ def test_10_write_starvation(self):
         # delete the key
         self.conn.delete(GRAPH_ID)
 
+    def test_11_concurrent_resize_zero_matrix(self):
+        self.graph = Graph(self.conn, GRAPH_ID)
+
+        self.graph.query("CREATE (:N)")
+
+        def resize_and_query():
+            g = redis.commands.graph.Graph(self.conn, GRAPH_ID)
```

Comment:
I believe you want to use a new connection here, as Redis handle a single command each time from each connection